Lua/Comments.tmPreferences

```diff
@@ -15,6 +15,18 @@
 				<key>value</key>
 				<string>-- </string>
 			</dict>
+			<dict>
+				<key>name</key>
+				<string>TM_COMMENT_START_2</string>
+				<key>value</key>
+				<string>--[[</string>
+			</dict>
+			<dict>
+				<key>name</key>
+				<string>TM_COMMENT_END_2</string>
+				<key>value</key>
+				<string>--]]</string>
```

Comment:
One of the similarities between Lua and Python is their use of the same symbols for multiline comments as for multiline strings (`'''` and `[[]]`).

In Lua, you start a multiline string with `[[`, and a comment with `--[[`, but both of them can be ended with just `]]`. The reasoning for implementing this as `--]]` instead is just to add a distinguishing factor to it so you can easily tell it's a comment ending and not a string ending.  
(I'm not going to bother with the fact that you can infix up to multiple `=` symbols in either)